release-notes/4.1.1.md

```diff
@@ -0,0 +1,26 @@
+# Release notes for Gluster 4.1.0
```

Comment:
I believe you intended this to be `4.1.1` ? Here and elsewhere through-out this file.

Reply:
I had this corrected in the glusterfs repos but copied over an older file, thanks for catching it. Corrected now!

Only the one instance needs correction, other instances point to the last minor/major release as per text.

File in glusterfs repo: https://github.com/gluster/glusterfs/blob/release-4.1/doc/release-notes/4.1.1.md